README.md

```diff
@@ -40,6 +41,25 @@ After cloning a repository, Macaron parses the CI configuration files and bash s
 
 To learn how to define your own checks, see the steps in the [checks documentation](/src/macaron/slsa_analyzer/checks/README.md).
 
+## Publications
+
+* Behnaz Hassanshahi, Trong Nhan Mai, Alistair Michael, Benjamin Selwyn-Smith, Sophie Bates, and Padmanabhan Krishnan: [Macaron: A Logic-based Framework for Software Supply Chain Security Assurance.](https://dl.acm.org/doi/abs/10.1145/3605770.3625213) SCORED 2023. (Best paper award)
```

Comment:
```suggestion
* Behnaz Hassanshahi, Trong Nhan Mai, Alistair Michael, Benjamin Selwyn-Smith, Sophie Bates, and Padmanabhan Krishnan: [Macaron: A Logic-based Framework for Software Supply Chain Security Assurance.](https://dl.acm.org/doi/abs/10.1145/3605770.3625213) SCORED 2023 (Best paper award).
```

Reply:
See commit [b9f10d2](https://github.com/oracle/macaron/pull/604/commits/b9f10d280cd32111818c190249c8a0206af74a34)

Reply:
I actually added the :trophy: code point and updated the copyright year :slightly_smiling_face: 
See commit [48bcc33](https://github.com/oracle/macaron/pull/604/commits/48bcc33d25aa3e3e86b818512cdac8fdfabbd6cb)